packages/vscode-extension/tests/fixtures/.env

```diff
@@ -1,6 +0,0 @@
-NEO4J_SCHEME=neo4j
-NEO4J_HOST=localhost
-NEO4J_PORT={PORT}
-NEO4J_USER=neo4j
-NEO4J_DATABASE=neo4j
-NEO4J_PASSWORD={PASSWORD}
```

Comment:
I realized we don't need this file anymore because it was used to pass the port and password to the tests, but we can do that via a better mechanism